### PR TITLE
Fix broken Charter link

### PR DESCRIPTION
**Pull request by imnmo**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: communication/marketing-team/storytelling-resources/social-guidelines.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## General guidelines
 
-All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](./CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
+All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
```
Comment (mrbobbytables) [inline communication/marketing-team/storytelling-resources/social-guidelines.md:7]:
```suggestion
All messaging must be consistent with the values and principles of the project and the [ethos/vision of the team](/communication/marketing-team/CHARTER.md#ethosvision). Messaging should be positive and uplifting. Be aware that sarcasm and/or jokes are generally hard to read over a medium such as social media.
```
For inter-repo documentation, a relative link is preferred 👍 
(they're not impacted by branch renames etc)

**Comment by imnmo** [comment]
folks, if anyone has time kindly review and need ok-to-test

**Comment by jberkus** [comment]
@imnmo the current link seems to work?  What is this fixing?

**Comment by imnmo** [comment]
@jberkus the current link points towards /marketing-team/storytelling-resources/CHARTER.md but it's been moved one level up to marketing-team/CHARTER.md 

here is the error when trying to access the link provided in the markdown

image

**Comment by jberkus** [comment]
Hmmm.  I don't get the error, but the new link works too.

/lgtm
/approve

**Pull request merged**